apps/oeth/src/components/ApyHeader.tsx

```diff
@@ -0,0 +1,236 @@
+    address,
+    token: tokens.mainnet.OUSD.address,
+    watch: true,
+  });
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <Menu
+        anchorEl={anchorEl}
+        open={!!anchorEl}
+        onClose={handleClose}
+        MenuListProps={{ dense: true }}
+      >
+        {days.map((day) => (
+          <MenuItem
+            divider
+            key={day}
+            selected={selectedPeriod === day}
+            onClick={() => {
+              setSelectedPeriod(day);
+              setAnchorEl(null);
+            }}
+          >
+            {intl.formatMessage(
+              { defaultMessage: '{days} day trailing' },
+              { days: day },
+            )}
+          </MenuItem>
+        ))}
+      </Menu>
+      <Stack
+        direction="row"
+        flexWrap="wrap"
+        gap={{ xs: 1, md: 2 }}
+        sx={{ width: '100%' }}
+      >
+        <Box
+          sx={{
+            paddingInline: { xs: 3, md: 2.5 },
+            paddingBlock: 2,
+            borderRadius: 1,
+            border: '1px solid',
+            borderColor: 'grey.800',
+            flexBasis: {
+              xs: '100%',
+              md: '25%',
+            },
+            display: 'grid',
+            placeContent: 'center',
+          }}
+        >
+          <Typography
+            color="text.secondary"
+            variant="body2"
+            alignItems="center"
+          >
+            {intl.formatMessage(
+              { defaultMessage: '{days} day trailing APY' },
+              { days: selectedPeriod },
+            )}
+          </Typography>
+          <Stack
+            direction="row"
+            alignItems="center"
+            sx={{ justifyContent: { xs: 'center', md: 'flex-start' } }}
+          >
+            <Typography
+              sx={{
+                fontWeight: 700,
+                fontSize: '1.5rem',
+                fontStyle: 'normal',
+                lineHeight: '2rem',
+                WebkitTextFillColor: 'transparent',
+                backgroundClip: 'text',
+                fontFamily: 'Sailec',
+                backgroundImage:
+                  'linear-gradient(90deg, var(--mui-palette-primary-light) 0%, #6A36FC 100%)',
+              }}
+            >
+              {intl.formatNumber(6.71 / 100, {
+                minimumFractionDigits: 2,
+                style: 'percent',
+              })}
+            </Typography>
+            <IconButton
+              disableRipple
+              onClick={(e) => setAnchorEl(e.currentTarget)}
+              sx={{
+                backgroundColor: (theme) =>
+                  alpha(theme.palette.common.white, 0.15),
+                marginInlineStart: 1,
+                alignSelf: 'center',
+                position: 'relative',
+                height: '26px',
+                borderRadius: '100%',
+                top: '-2px',
+              }}
+            >
+              <Box
+                component={'img'}
```

Comment:
Minor nitpick, I assume for anything is just a string w/ no dynamic variables, we will opt to remove `{ }`